src/main/java/Steps/MyStepdefs.java

```diff
@@ -35,31 +37,32 @@ public void setup() {
 
     @Given("^User is created$")
     public void userIsCreated() throws Throwable {
-        Post user1 = new Post(23,"test","test","test","test","test");
-
-
-        GsonBuilder builder = new GsonBuilder();
-        Gson gson = builder.create();
-
+        long currentTimestamp = System.currentTimeMillis();
+        User user1 = new User(
+                "AUT_name_" + currentTimestamp,
+                "AUT_username_" + currentTimestamp,
+                "AUT_email_" + currentTimestamp,
+                "AUT_address_" + currentTimestamp,
+                ""+currentTimestamp
+        );
+
+        gson = builder.create();
         String jsonInString = gson.toJson(user1);
 
         HttpResponse<JsonNode> response
                 = Unirest.post("https://seleniumclass.000webhostapp.com/api/v1/users")
+                .header("Content-Type", "application/json")
                 .body(jsonInString)
                 .asJson();
 
+        myUser = gson.fromJson(response.getBody().toString(), User.class);
```

Comment:
here i deserialize response - in response I have id of newly created user, so i can use it